autocomplete_light/autocomplete/model.py

```diff
@@ -44,6 +43,15 @@ class AutocompleteModel(object):
         single field name or an iterable (ie. list, tuple).
         However, if AutocompleteModel is instanciated with a list of values,
         it'll reproduce the ordering of values.
+
+    .. py:attribute:: ordering_alias
+
+        Internal attribute that determines which alias is used by
+        injected sql that will show selected items first. Default is
+        _ordering that is probably very unusual as field_name. You
+        shuld worry to set this attribute only if you have an
+        attribute on db table named '_ordering'
```

Comment:
Does it really make sense to change the "ordering" alias here ? Shouldn't we just use a less-conflicting name like `_autocomplete_light_ordering_` or something like that ?